examples/edge-stack-iso/kubernetes/manifests/neuvector-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    pod-security.kubernetes.io/enforce: privileged
```

Comment:
https://github.com/neuvector/neuvector-helm?tab=readme-ov-file#deploy-in-kubernetes